Move include out of function to avoid error:
/usr/include/sys/stat.h:453:1: error: nested function ‘stat’ declared ‘extern’
Author: Samuel Moors (Vrije Universiteit Brussel)
diff -Nur corsika-77550.orig/src/parallel/mpi_runner.c corsika-77550/src/parallel/mpi_runner.c
--- corsika-77550.orig/src/parallel/mpi_runner.c	2024-04-18 18:30:39.000000000 +0200
+++ corsika-77550/src/parallel/mpi_runner.c	2024-08-09 16:15:39.969688000 +0200
@@ -99,6 +99,7 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include "config.h"
+#include <sys/stat.h>
 
 /////////////////////initializing parameters/////////////////// 
 //the number of data type block in the MPI message
@@ -1023,7 +1024,6 @@
               strcpy(str2, strtmp);
             }
 						strcpy(statdir,str2);
-						#include <sys/stat.h>
 						struct stat sb;
 						if (stat(statdir, &sb) == 0 && S_ISDIR(sb.st_mode))
 						{
